src/components/GlobalNav/GlobalNav.jsx

```diff
@@ -1,7 +1,55 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import { Link } from 'react-router-dom';
+import './GlobalNav.scss';
 
-function GlobalNav(props) {
-  return <div>NavBar</div>;
+function GlobalNav() {
+  const [scrollPosition, setScrolPosition] = useState(0);
+  const updateScrollPosition = () => {
+    setScrolPosition(window.scrollY || document.documentElement.scrollTop);
+  };
+  useEffect(() => {
+    window.addEventListener('scroll', updateScrollPosition);
+  });
+  return (
+    <header className="GlobalNav">
+      <div className={scrollPosition < 1 ? 'originalNav' : 'miniNav'}>
+        <Link to="/">
+          <div className="logoWrapper">
+            <span>MOTHER TERAROSA</span>
+          </div>
+        </Link>
+        <Link to="/shoplist/:category">
+          <div className="categoryWrapper">
+            <div>SHOP</div>
+          </div>
+        </Link>
```

Comment:
```suggestion
        <Link to="/" className="logoWrapper">
          MOTHER TERAROSA
        </Link>
        <Link to="/shoplist/:category" className="categoryWrapper">
          SHOP
        </Link>
```
이런식으로 불필요한 depth는 줄여주세요!